lib/fetchers/fetcher.py

```diff
@@ -18,12 +16,16 @@
 from dplaingestion.selector import setprop
 from dplaingestion.selector import getprop as get_prop
 from dplaingestion.utilities import iterify, couch_id_builder
+import requests
+import re
```

Comment:
I've submitted https://github.com/dpla/ingestion/pull/128 to address this import issue.